### PR TITLE
Repository creation guidelines

### PR DESCRIPTION
**Pull request by flyingfrog81**
This PR includes all the documentation related to the creation of new repositories.

### CODE REVIEW

**Comment by flyingfrog81** [comment]
This is a first stub and I'm welcome to iterate based on this version. Please push straight into the branch if you see fit.

File: src/projects/create_new_project.rst [hunk 1]
```diff
@@ -1,6 +1,70 @@
...
+
+Mono VS Multi repositories
+==========================
+
+One of the first choices when creating a new project is how to split the code into repositories. 
+In a project such as SKA there is no strict rule that can be applied, and a degree of judgement is 
+necessary from the developers.
+In general, a repository should be created for every software component following this definition: 
+
+  *All software and firmware source code handed over to the SKA organisation shall be organised into source code repositories. A source code repository is a set of files and metadata, organized in a directory structure. It is expected that source code repositories map to individual applications or modules according to the following definition: A component is reusable, replaceable with something else that implements the same API, independently deployable, and encapsulates some coherent set of behaviors and responsibilities of the system.*
```
Comment (Matteo04052017) [inline src/projects/create_new_project.rst]: Well, there is a difference between component and module. 
I agree on source code repositories that map to individual applications or modules but those are not components. 
In tango-controls, a component is a device or a device server and if we go for this definition then we will end up with too many repositories. The more repositories we have, the more complicated the integration would be. 
I would prefer to have one repository per module and for each module one team.

**Review by Matteo04052017 — CHANGES_REQUESTED** [review]
I would change the rule for creating a repository... the map should be 1 Module - 1 Repository where the module is what you assign to a particular team.

**Comment by flyingfrog81** [comment]
I would say that this could be Many modules one Repo, as long as the modules expose a self-included behavior.

**Comment by flyingfrog81** [comment]
Merging this as no feedback is provided

**Pull request merged**